pigeon.py

```diff
@@ -34,7 +34,7 @@
 
 logging.config.dictConfig({
     'version': 1,
-    'disable_existing_loggers': True,
+    # 'disable_existing_loggers': True,
```

Comment:
I should remove this and add a comment about why this typically-there line isn't.